css/style.css

```diff
@@ -142,18 +142,35 @@ p > a {
   margin-left: 10px;
 }
 
+/*I'm only seeing wrap-left being used on the About page but it's listed under .team
```

Comment:
This seems good. We could probably clean up our .team css and stuff. For example, looks like I've got the team class in volunteers still, but not sure it's really using it. But things are looking good for what we've got, so I'm good with it for getting the site out there.